mods/ts/rules/defaults.yaml

```diff
@@ -339,8 +304,50 @@
 		UpgradeTypes: hospitalheal
 		UpgradeMinEnabledLevel: 1
 
+^RegularInfantryDeath:
+	WithDeathAnimation@normal:
+		DeathTypes:
+			BulletDeath: 1
```

Comment:
Could we change these to use descriptive death suffixes instead of integers? I'm assuming that you only need to change a handful of places now that you've deduplicated both the rules and sequence setup.